gpcontrib/arenadata_toolkit/arenadata_toolkit--1.2--1.3.sql

```diff
@@ -0,0 +1,55 @@
+/* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.2--1.3.sql */
+
+/*
+ * Returns column pair (table_schema, table_name) ordered by haven't been vacuumed
+ * before at the head of the list and vacuumed before by ascending order (default strategy)
+ */
+CREATE FUNCTION arenadata_toolkit.adb_vacuum_strategy_newest_first(actionname TEXT)
+RETURNS TABLE (table_schema NAME, table_name NAME) AS
+$$
+BEGIN
+	RETURN query
+	SELECT n.nspname AS table_schema, c.relname AS table_name
+	FROM pg_class c
+	JOIN pg_namespace n ON c.relnamespace = n.oid
+	LEFT JOIN pg_stat_last_operation o ON o.classid = 'pg_class'::regclass::oid AND o.objid = c.oid AND o.staactionname = actionname
```

Comment:
`o.staactionname = actionname`. This does not work when `actionname` is lowercase:
```
SELECT n.nspname AS table_schema, c.relname AS table_name, o.staactionname
FROM pg_class c
JOIN pg_namespace n ON c.relnamespace = n.oid AND c.relname='vacuumed'
LEFT JOIN pg_stat_last_operation o ON o.classid = 'pg_class'::regclass::oid AND o.objid = c.oid 
AND o.staactionname = 'VACUUM';

 table_schema | table_name | staactionname 
--------------+------------+---------------
 test_vacuum  | vacuumed   | VACUUM
(1 row)

SELECT n.nspname AS table_schema, c.relname AS table_name, o.staactionname
FROM pg_class c
JOIN pg_namespace n ON c.relnamespace = n.oid AND c.relname='vacuumed'
LEFT JOIN pg_stat_last_operation o ON o.classid = 'pg_class'::regclass::oid AND o.objid = c.oid 
AND o.staactionname = 'vacuum';
 table_schema | table_name | staactionname 
--------------+------------+---------------
 test_vacuum  | vacuumed   | 
(1 row)
```

Reply:
> This does not work when `actionname` is lowercase

Yes, as in original query

Reply:
I.E it's the expected behaviour?

Reply:
Fixed